web/templates/web/study-detail.html

```diff
@@ -2,233 +2,160 @@
+                            <h4>{% trans "What We're Studying" %}</h4>
+                        </td>
+                        <td class="{% study_info_table_content_classes %}">{{ object.purpose|linebreaks }}</td>
+                    </tr>
+                    <tr>
+                        <td class="{% study_info_table_heading_classes %}">
+                            <h4>{% trans "Duration" %}</h4>
+                        </td>
+                        <td class="{% study_info_table_content_classes %}">{{ object.duration|linebreaks }}</td>
+                    </tr>
+                    {% if object.compensation_description %}
+                        <tr>
+                            <td class="{% study_info_table_heading_classes %}">
+                                <h4>{% trans "Compensation" %}</h4>
+                            </td>
+                            <td class="{% study_info_table_content_classes %}">{{ object.compensation_description|linebreaks }}</td>
+                        </tr>
+                    {% endif %}
+                </table>
+                <p class="pb-0">
+                    <em class="mt-3 mb-0 text-center text-muted">{% trans "This study is conducted by" %} {{ object.contact_info }}</em>
+                </p>
+            </div>
+        </div>
+        <div class="col-md-3">
+            <h4 class="mt-4 mb-3">{% trans "Would you like to participate in this study?" %}</h4>
+            {% if not request.user.is_authenticated %}
+                <div class="my-3">{% nav_login request text="Log in to participate" button=True %}</div>
+                <div class="my-3">{% nav_signup request text="Create a new account" button=True %}</div>
+            {% elif not children %}
+                {% bootstrap_button add_child_profile href=add_child_url button_class=btn_secondary_classes %}
+            {% elif not has_demographic %}
+                {% bootstrap_button complete_demo href=complete_demo_url button_class=btn_secondary_classes %}
+            {% else %}
+                <div class="form-group">
+                    <form method="post">
+                        {% csrf_token %}
+                        <label for="child-dropdown" class="form-label">{% trans "Select a child:" %}</label>
+                        <select id="child-dropdown"
+                                name="child_id"
+                                class="form-select"
+                                data-study-min-age-days="{{ study.min_age_days }}"
+                                data-study-min-age-months="{{ study.min_age_months }}"
+                                data-study-min-age-years="{{ study.min_age_years }}"
+                                data-study-max-age-days="{{ study.max_age_days }}"
+                                data-study-max-age-months="{{ study.max_age_months }}"
+                                data-study-max-age-years="{{ study.max_age_years }}"
```

Comment:
These study age criteria values were previously used in the JavaScript that was embedded on the template page. Now that the JavaScript is in a separate file, these values can't be accessed directly so I'm passing through as data here.